**Práctica 4**

1. El propietario de Britten’s Egg Farm desea calcular la cantidad media de
huevos que pone cada gallina. Una muestra de 20 gallinas indica que
ponen un promedio de 20 huevos al mes, con una desviación estándar de
2 huevos al mes.\
a) ¿Cuál es el valor de la media de la población? ¿Cuál es el mejor estimador de
este valor?\
b) Explique por qué necesita utilizar la distribución t. ¿Qué suposiciones necesita
hacer?\
c) ¿Cuál es el valor de t en un intervalo de confianza de 95%?\
d) Construya el intervalo de confianza de 95% de la media de población.\
e) ¿Es razonable concluir que la media poblacional es de 21 huevos? ¿Y de 25
huevos?

In [10]:
import numpy as np
import scipy.stats as st

n = 20
media_muestral = 20
nivel_confianza = 0.95
s = 2

t_crit = st.t.ppf((1+nivel_confianza)/2, df=n-1)
print(f'EL VALOR DE t ES: {t_crit}')
ic_inferior = media_muestral - t_crit * (s / np.sqrt(n))
ic_superior = media_muestral + t_crit * (s / np.sqrt(n))

print(f'Intervalo de confianza 95%: [ {ic_inferior} - {ic_superior} ]')


EL VALOR DE t ES: 2.0930240544083087
Intervalo de confianza 95%: [ 19.06397118716016 - 20.93602881283984 ]


**RESPUESTAS**

- a. No conocemos el valor de la media poblacional, el mejor estimador por lo tanto seria la media muestral
- b. Tambien se desconoce la desviacion estandar de la poblacion, y nuestro tamaño de muestra es menor a 30, asi que
- c. El valor t es _**t=2.093**_
- d. Los intervalos de confianza son: _**[ 19.06397118716016 - 20.93602881283984 ]**_
- e. No ya que ambos valores estan fuera de los intervalos de confianza obtenidos




2. Sean las siguientes hipótesis: 𝐻0: 𝜇 = 100; 𝐻1: 𝜇 ≠ 100 , una muestra
aleatoria de seis elementos dio como resultado los siguientes valores: 118,
105, 112, 119, 105 y 111. ¿Puede concluir que la media poblacional es
diferente de 100 con un nivel de significancia de 0.05?

a) Establezca la regla de decisión.

b) Calcule el valor del estadístico de prueba.

c) ¿Cuál es su decisión en lo que se refiere a la hipótesis nula?

**RESPUESTAS**
- a.

In [3]:
data = np.array([118,105,112,119,105,111])

n = len(data)
media_muestral = np.mean(data)
s = np.std(data, ddof=1)

t = (media_muestral - 100) / (s / np.sqrt(n))

print("Media:", media_muestral)
print("Desviación:", s)
print("t calculado:", t)


Media: 111.66666666666667
Desviación: 6.0553007081949835
t calculado: 4.719399037242696


3. La familia Damon posee un viñedo grande en el oeste de Nueva York a orillas de lago
Erie. Los viñedos deben fumigarse al inicio de la temporada de cultivo para protegerlos
contra diversos insectos y enfermedades. Dos nuevos insecticidas acaban de salir al
mercado: Pernod 5 y Action. Para probar su eficacia, se seleccionaron tres hileras y se
fumigaron con Pernod 5, y otras tres se fumigaron con Action. Cuando las uvas
maduraron, se revisaron 400 vides tratadas con Pernod 5 para saber si no estaban
infectadas. De igual forma, se revisó una muestra de 400 vides fumigadas con Action.
Los resultados son:
Con un nivel de significancia de 0.05, ¿se puede concluir que existe una diferencia entre
la proporción de vides infectadas empleando Pernod 5 en comparación con las
fumigadas con Action? Para el siguiente ejercicio

a) formule la regla de decisión,

b) calcule la estimación conjunta de la varianza de la población

c) calcule el estadístico de
prueba,

d) tome una decisión respecto de la hipótesis nula

In [4]:
n1, x1 = 400, 24
n2, x2 = 400, 40

p1 = x1/n1
p2 = x2/n2

p_pool = (x1+x2)/(n1+n2)

z = (p1-p2)/np.sqrt(p_pool*(1-p_pool)*(1/n1+1/n2))

print("z:", z)

z: -2.085144140570748
